backend/common/src/purl.rs

```diff
@@ -128,15 +128,18 @@ impl Debug for Purl {
 
 impl From<&str> for Purl {
     fn from(value: &str) -> Self {
-        PackageUrl::from_str(value).unwrap().into()
+        match PackageUrl::from_str(value) {
+            Ok(s) => s.into(),
+            Err(e) => Default::default(),
+        }
     }
 }
 
-impl From<&&str> for Purl {
-    fn from(value: &&str) -> Self {
-        PackageUrl::from_str(value).unwrap().into()
-    }
-}
+// impl From<&&str> for Purl {
+//     fn from(value: &&str) -> Self {
+//         PackageUrl::from_str(value).unwrap().into()
+//     }
+// }
```

Comment:
@bobmcwhirter I thought this looked weird. It's not used, but I forgot to delete it.